sn_node/src/put_validation.rs

```diff
@@ -700,6 +700,23 @@ impl Node {
                 }
                 spends
             }
+            Err(NetworkError::GetRecordError(GetRecordError::NotEnoughCopies {
```

Comment:
I am not sure, but this sounds bit tricky.
a cheaty node can easily provide one copy of a shaky spend, and this means any query attempt to that spend will replied with `NotEnoughCopies`, then be put into further handling.
This actually sounds like an `attack window` opened up ?

Reply:
Yeah, Roland, when testing locally, do you often see the following log line??